flexget/plugins/plugin_rarfile.py

```diff
@@ -0,0 +1,196 @@
+from __future__ import unicode_literals, division, absolute_import
+import logging
+import os
+import rarfile
```

Comment:
I think it'd be best to have some conditional importing so as not to break everything if `rarfile` is not installed.